tests/album-delete.test.js

```diff
@@ -0,0 +1,65 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Delete album', () => {
+  let artists;
+  let albums;
+  let album;
+  beforeEach(async () => {
```

Comment:
As above its a good idea to add a clear line between the different parts of your test, in this case the variable creation and the `beforeEach` function
```
  let album;

  beforeEach(async () => {
```